openlibrary/plugins/admin/code.py

```diff
@@ -431,10 +441,15 @@ def revert(self, changeset_ids, comment):
             for cid in changeset_ids
             for c in site.get_change(cid).changes
         ]
-
+        docs = [
+            doc for doc in docs if doc.get('type', {}).get('key') != '/type/delete'
+        ]
```

Comment:
```suggestion
        docs = [doc for doc in docs if doc.get('type', {}).get('key') != '/type/delete']
```
This is the result of running `black openlibrary/plugins/admin/code.py`